authorization/README.md

```diff
@@ -6,4 +6,4 @@
 > [!NOTE]  
 > Adding your username to one of the `Group`s managed by this ArgoCD application does **not** create your username or grant access to the cluster. You must already have an internal SSO login in order for these role bindings to take effect.
 
-Membership in the `redhat-ai-dev-users` `Group` on the cluster grants cluster-wide `edit` permissions. Additional permissions can be configured via additional `Group` and `RoleBinding` resources.
\ No newline at end of file
+Membership in the `redhat-ai-dev-users` `Group` on the cluster grants cluster-wide `view` and `rhoai-workspace` namespace `edit` permissions. Additional permissions can be configured via additional `Group` and `RoleBinding` resources.
```

Comment:
One minor nit: Let's also clarify that it gives them a personal user namespace which they'll have edit permissions in.